src/frontend/sider/Fagsakoversikt.tsx

```diff
@@ -26,44 +33,68 @@ const TittelWrapper = styled.div`
-        [fagsakId]
-    );
+        }).then((response) => settFagsak(response));
 
-    const fagsak = useDataHenter<Fagsak, null>(a);
+    const hentPersonData = () =>
+        axiosRequest<IPersonopplysninger, null>({
+            method: 'GET',
+            url: `/familie-ef-sak/api/personopplysninger/fagsak/${fagsakId}`,
+        }).then((response) => settPersonOpplysninger(response));
 
-    return (
-        <DataViewer response={fagsak}>
-            {(data) => {
-                return (
-                    <>
-                        <VisittkortWrapper className="blokk-s">
-                            <Visittkort
-                                alder={19}
-                                ident="007"
-                                kjønn={kjønnType.MANN}
-                                navn="Batman"
-                            />
-                        </VisittkortWrapper>
-                        <TittelWrapper>
-                            <Innholdstittel className="blokk-m" tag="h2">
-                                Behandlingsoversikt - Batman{' '}
-                            </Innholdstittel>
-                            <Systemtittel tag="h3">
-                                Fagsak: {formatterEnumVerdier(data.stønadstype)}
-                            </Systemtittel>
-                        </TittelWrapper>
-                        <FagsakoversiktTabell behandlinger={data.behandlinger} />
-                    </>
-                );
-            }}
-        </DataViewer>
-    );
+    useEffect(() => {
+        if (fagsakId) {
+            hentFagsak();
+            hentPersonData();
+        }
+    }, [fagsakId]);
+
+    if (erAvTypeFeil(fagsak) || erAvTypeFeil(personOpplysninger)) {
+        return (
+            <AlertStripeFeil>
+                <p>{(fagsak as RessursFeilet).frontendFeilmelding ?? ''}</p>
+                <p>{(personOpplysninger as RessursFeilet).frontendFeilmelding ?? ''}</p>
+            </AlertStripeFeil>
+        );
+    }
+
+    if (
+        fagsak.status === RessursStatus.SUKSESS &&
+        personOpplysninger.status === RessursStatus.SUKSESS
+    ) {
+        return (
+            <>
+                <VisittkortWrapper className="blokk-s">
+                    <Visittkort
+                        alder={20}
+                        ident={personOpplysninger.data.personIdent}
+                        kjønn={personOpplysninger.data.kjønn}
+                        navn={personOpplysninger.data.navn.visningsnavn}
+                    />
+                </VisittkortWrapper>
+                <TittelWrapper>
+                    <Innholdstittel className="blokk-m" tag="h2">
+                        Behandlingsoversikt - {personOpplysninger.data.navn.visningsnavn}
+                    </Innholdstittel>
+                    <Systemtittel tag="h3">
+                        Fagsak: {formatterEnumVerdi(fagsak.data.stønadstype)}
+                    </Systemtittel>
+                </TittelWrapper>
```

Comment:
Blir personlinjen bare på fagsakoversikten eller for alle behandlingsstegene nå? Tror vi vil ha den på alle behandlingssidene iallefall

Reply:
[b6aec6e](https://github.com/navikt/familie-ef-sak-frontend/pull/106/commits/b6aec6e0216fad4e0f15d81639f4866f674f4ba0)